src/hooks/useGetStagingStatus.ts

```diff
@@ -14,6 +14,7 @@ export const useGetStagingStatus = (
     () => getStagingBuildStatus({ siteName }),
     {
       retry: false,
+      refetchInterval: 1000 * 5, // 5 sec for quicker feedback when user press save
```

Comment:
Will we encounter any rate limits by Amplify?

Reply:
completely fair, their documention dont have. 
1. asking aws folks about this 
2. throttle the release first